mlflow/xgboost.py

```diff
@@ -294,16 +298,13 @@ def autolog(importance_types=["weight"]):  # pylint: disable=W0102
     Note that the `scikit-learn API`_ is not supported.
 
     :param importance_types: importance types to log.
-
+    :param log_input_example: whether to log a sample of the training data as an example for future
```

Comment:
Can we clarify that the example is logged as part of the model? (i.e., as opposed to a tag / parameter)